server/graphql/v2/object/Order.js

```diff
@@ -104,6 +105,14 @@ export const Order = new GraphQLObjectType({
           return { value, currency: order.currency };
         },
       },
+      // needed for recurring contributions work, but we should update to encoded id and write v2 payment method object soon
+      paymentMethod: {
+        type: PaymentMethod,
+        async resolve(order) {
+          const orderPM = await models.PaymentMethod.findByPk(order.PaymentMethodId);
+          return orderPM;
```

Comment:
Now you can simply `return models.PaymentMethod.findByPk(order.PaymentMethodId);`